deployment/dev_env/docker-compose.yml

```diff
@@ -1,12 +1,5 @@
 version: "2"
 
-x-logging:
```

Comment:
Can we reuse this config? You don't need x-logging, you can do it as shown here: https://stackoverflow.com/questions/38567355/docker-compose-global-level-logging